deprecations/index.js

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Is there a reason we need a `deprecations` directory with only one file in it? Are we planning to add more things to this directory?